### ISSUE TITLE
`prompt` in `~/.myclirc` is completely ignored after updating to 1.27.1.

### ISSUE THREAD

**Issue by your-diary** [dbcli/mycli#1155]
```
$ mycli --version
Version: 1.27.1

$ grep prompt ~/.myclirc
prompt = '\n\A (\d) > '

$ mycli -D main #`prompt` in `~/.myclirc` is ignored.
MySQL root@localhost:t>

$ mycli --prompt hello_world -D main #`--prompt` is respected though.
hello_world
```

I don't have this problem in 1.27.0.

Maybe due to

- #1145

**Comment by your-diary** [comment]
Reverting #1145 fixed the problem.

**Comment by amjith** [comment]
Thanks for reporting it. I'll try and reproduce it.

**Comment by otooi** [comment]
I experienced the same symptoms.
When I rolled back #1144 as @your-diary said, I was able to use the prompt normally.
The prompt I used was this.
`prompt = '\u@\h:\p:\d\_>\n'`

**Comment by hzc989** [comment]
sorry to produce the bug and bother you guys [lol] , I recently encountered that and found this issue.

**Comment by amjith** [comment]
@hzc989 What version are you using? This is fixed in 1.27.2
